src/neuroconv/tools/testing/mock_interfaces.py

```diff
@@ -128,7 +128,8 @@ def __init__(
         self,
         num_channels: int = 4,
         sampling_frequency: float = 30_000.0,
-        durations: List[float] = [1.0],
+        # durations: List[float] = [1.0],  # Uncomment when pydantic is integrated for schema validation
+        durations: list = [1.0],
```

Comment:
Maybe remove the comment, also ,didn't you wanted to make this tuples? We can cast to list inside or do the None with if.

Reply:
Good catch, swapped to tuple

This is related to how this interfaces was never tested in a `NWBConverter` which does additional validation than single interface write